src/poprox_recommender/components/diversifiers/locality_calibration.py

```diff
@@ -34,16 +35,20 @@ def __call__(self, candidate_articles: ArticleSet, interest_profile: InterestPro
 
         article_scores = article_scores.cpu().detach().numpy()
 
-        article_indices = self.calibration(
+        article_indices, final_calibrations = self.calibration(
             article_scores,
             candidate_articles.articles,
             normalized_topic_prefs,
             normalized_locality_prefs,
-            self.theta_topic,
-            self.theta_local,
+            theta_topic,
+            theta_locality,
             topk=self.num_slots,
         )
 
+        # Save computed kl divergence for topic and locality
+        with open(KL_VALUE_PATH, 'a') as file:
```

Comment:
Can we set a flag to whether we open/write to file? Otherwise we'll have to rip this out later when we try to deploy to a server.